src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java

```diff
@@ -37,6 +37,20 @@ public static void main(String[] args) {
     static String getGreetingByHour(int hour) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String greatingUser;
+
+        if (hour > 5 && hour < 12) {
+            greatingUser = "Good morning";
+        } else if (hour >= 12 && hour < 18) {
+            greatingUser = "Good day";
+        } else if (hour >= 18 && hour < 24) {
+            greatingUser = "Good evening";
+        } else if (hour >= 0 && hour < 6){
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"

Reply:
fixed